nx/source/services/ncm.c

```diff
@@ -27,14 +27,30 @@ static Result _ncmGetInterfaceInU8(Service* srv_out, u32 cmd_id, u8 inval) {
     );
 }
 
-static Result _ncmCmdNoIo(Service* srv, u32 cmd_id) {
+NX_INLINE Result _ncmCmdNoIo(Service* srv, u32 cmd_id) {
```

Comment:
Leave these at static, defeats the point of having dedicated funcs instead of calling serviceDispatch*() directly otherwise (smaller code size).